nestjs/.eslintrc.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'tsconfig.json',
+    sourceType: 'module',
+  },
+  plugins: ['@typescript-eslint/eslint-plugin'],
+  extends: [
+    'plugin:@typescript-eslint/recommended',
```

Comment:
Could we use the same as for angular ? If need ask me /
Please synchronise  with Gleb and lets meet together